app/components/SocketProvider.js

```diff
@@ -287,6 +290,16 @@ class SocketProvider extends React.Component<Props> {
       }
     });
 
+    this.socket.on("collections.export_all", (event) => {
+      const user = auth.user;
+      if (user) {
+        exports.data.set(event.id, {
```

Comment:
Could we just use the eventPresenter here?

Reply:
I'm afraid Idk what do you mean by `exportPresenter`?

Reply:
presenters/export

Reply:
Hmm true, I misread this – but then I don't really understand why we're spreading the current authenticated user here, adding the user data that triggered the export would be a server-side concern. That's what threw me 😄

Reply:
I have learnt the hard way in the past to not have different places serializing the same backend model, as our typing gets better it'll also be harder to type these payloads that are half built.

Reply:
I will keep it in mind. Not doing the spreading anymore.